site/guide/manage-users.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Manage users"
+keywords: "manage users, user management, ValidMind"
+---
+
+Managing users is fundamental to maintaining access control and efficient user management in the ValidMind system. This task topic will guide you through the process of inviting new users, assigning them to specific groups, and configuring their roles. Whether you're a group administrator or a superuser, understanding these steps is crucial for ensuring the right individuals have the appropriate access and permissions within the system.
+
+## Viewing your account profile
+1. Log in to the ValidMind UI.
+
+2. Go to Settings at the sidebar and select  **Profile** at Your Account.
```

Comment:
```suggestion
2. Go to **Settings** in the sidebar and select **Profile**.
```

This line had some issues, so I suggested the most likely change but please verify it's actually correct.